.lintstagedrc.json

```diff
@@ -0,0 +1,8 @@
+{
+  "*.{html,js,jsx,cjs,mjssvelte,ts,tsx,cts,mts,xml,yaml,yml}": [
+    "yarn buildDeps",
+    "prettier --write",
+    "eslint --fix --flag unstable_config_lookup_from_file"
```

Comment:
The flag allows us to use the new flat-style eslint config files in subdirectories the same way as the deprecated .eslintrc format. This will become the default behaviour with eslint 10.